src/ui/components/home/home_activity.rs

```diff
@@ -1,154 +1,615 @@
+    
+        let endOfPage = h + o >= b;
+    
+        // let iw = window().inner_width().ok().map(|b| b.as_f64().unwrap_or(0.0)).unwrap_or(0.0);
+        // let ow = window().outer_width().ok().map(|b| b.as_f64().unwrap_or(0.0)).unwrap_or(0.0);
+    
+          // let endOfPage: Result<bool, _> = { Ok(window().inner_height()?.into() + window().page_y_offset()?.into() >= document().body()?.offset_height().into()) };
+
+        logging::log!("{} {} {} {}", endOfPage, h, o, b);
+
+        if /* iw < 640f64 && */ endOfPage {
+    
+          // logging::log!("scroll {} width {} {}", endOfPage, iw, ow);
+    
+          // let ssr_posts = 
+          create_local_resource(
+            move || (user.get(), list_func(), sort_func()),
+            move |(_user, list_type, sort_type)| async move {
+              // let f = {
+              //   if let Some(t) = from.clone() {
+              //     if !t.is_empty() {
+              //       Some(PaginationCursor(t))
+              //     } else {
+              //       None
+              //     }
+              //   } else {
+              //     None
+              //   }
+              // };
+        
+              let form = GetPosts {
+                type_: list_type,
+                sort: sort_type,
+                community_name: None,
+                community_id: None,
+                page: None,
+                // limit: if csr_paginator.get().is_none() { Some(40) } else { None },
+                limit: None,
+                saved_only: None,
+                disliked_only: None,
+                liked_only: None,
+                page_cursor: csr_paginator.get(),
+              };
+        
+              let result = LemmyClient.list_posts(form).await;
+        
+              match result {
+                Ok(mut o) => {
+                  logging::log!("{:#?} page {:#?} ", csr_paginator.get(), o.next_page.clone());
+
+                  // if csr_paginator.get().is_none() {
+                    csr_paginator.set(o.next_page);
+                    let mut p = csr_posts.get().unwrap_or(vec![]);
+                    p.append(&mut o.posts);
+                    logging::log!("count {}", p.len());
+                    csr_posts.set(Some(p));
+                    logging::log!("{:#?} ua ", csr_paginator.get());
+                    // csr_posts.set(Some(csr_posts.get().unwrap_or(vec![]).append(o.posts)));
+                  // } else {
+                    // csr_paginator.set(o.next_page.clone());
+                  //   csr_posts.set(csr_posts.get());
+                  // }
+                  // Some(o)
+                },
+                Err(e) => {
+                  error.set(Some(e));
+                  // None
+                }
+              }
+            },
+          );
+      
+        }
+      };
+    
+      logging::log!("5");
+
+      window_event_listener_untyped("scroll", on_scroll);
+      // need resize hook as well
+    }
```

Comment:
What is all of this doing?

Reply:
i should have marked the PR as draft again before pushing update - this code may have been removed or changed